platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/GuestClaimsHandlerExt.java

```diff
@@ -108,6 +108,7 @@ public void init() {
             this.profiles = (Map<String, Object>) fromJson(json);
         } catch (IOException e) {
             LOGGER.debug("Could not find profiles.json during installation: ", e);
+            this.profiles = new HashMap<>();
```

Comment:
Curious why you chose not to initialize `profiles` like this in the constructor rather than initializing to `null` in the constructor?